Examples/php/cpointer/runme.php

```diff
@@ -6,35 +6,35 @@
 
 	print "Testing the pointer library\n";
 
-	$a = example::new_intp();
-	$b = example::new_intp();
-	$c = example::new_intp();
-	example::intp_assign($a,37);
-	example::intp_assign($b,42);
+	$a = new_intp();
+	$b = new_intp();
+	$c = new_intp();
+	intp_assign($a,37);
+	intp_assign($b,42);
```

Comment:
Similarly here - as we discussed on IRC, let's keep the `example` wrapper class in the generated `.php` wrapper to handle this case.